funnel/models/commentvote.py

```diff
@@ -277,7 +277,13 @@ def delete(self):
     @state.transition(None, state.SPAM)
     def mark_spam(self):
         """
-        Mark this comment as spam.
+        Mark this comment as spam
```

Comment:
RST guidelines for docstrings require the first line to be a single statement that ends in a period. Counter to our own convention of not using periods.